packages/controls/src/widget_audio.ts

```diff
@@ -5,19 +5,18 @@ import { DOMWidgetView } from '@jupyter-widgets/base';
 
 import { CoreDOMWidgetModel } from './widget_core';
 
-import * as _ from 'underscore';
-
 export class AudioModel extends CoreDOMWidgetModel {
   defaults(): Backbone.ObjectHash {
-    return _.extend(super.defaults(), {
+    return {
+      ...super.defaults(),
```

Comment:
The disadvantage to this is that it creates a new object at every level of the class hierarchy, every time a class is created. The other would create one defaults dictionary at the root and modify it for each subclass.

However, perhaps that was just premature optimization and we shouldn't worry about it.

Reply:
If this really does become an issue, we can use `Object.assign` directly instead of underscore these days.